std/os/linux/tls.zig

```diff
@@ -118,6 +118,9 @@ pub fn setThreadPointer(addr: usize) void {
                 : [addr] "r" (addr)
             );
         },
+        .arm => |arm| {
+            _ = std.os.linux.syscall1(std.os.linux.SYS_set_tls, addr);
```

Comment:
Should this check for an error and `@panic`?

Reply:
Not as far as i'm aware. I've found no documentation for the syscall, and in the linux source it always returns 0:
https://github.com/torvalds/linux/blob/5ad18b2e60b75c7297a998dea702451d33a052ed/arch/arm/kernel/traps.c#L638-L640

Reply:
All syscalls can return errors via e.g. `seccomp` shenanigans.